Exposing internal state via Kedro Viz React component props (#1969)
kedro-org/kedro-viz: RELEASE.md

- Display published URLs. (#1907)
- Conditionally move session store and stats file to .viz directory. (#1915)
- Refactor namespace pipelines. (#1897)

> - Expose internal Redux state via Kedro-Viz React component props `options`. (#1969)

Reviewer: Expose the internal Redux state via `options` in the Kedro-Viz React component.

> - Expose the internal Redux state via `options` in the Kedro-Viz React component. (#1969)

Reviewer 2: ```suggestion
- Expose the internal Redux state through `options` prop while using Kedro-Viz as a React component. (#1969)
```
